subprojects/docs/src/docs/release/notes.md

```diff
@@ -19,7 +19,7 @@ This allows processors with extension mechanisms to check extensions for increme
 
 New native plugins continue to improve with [better control over system include path](#better-control-over-system-include-path-for-native-compilation) for native compilation and [other improvements](https://github.com/gradle/gradle-native/blob/master/docs/RELEASE-NOTES.md#changes-included-in-gradle-48). 
 
-This release introduces 2 new APIs that facilitate immutable tasks: APIs to create [tasks with constructor arguments](#create-tasks-with-constructor-arguments) and [immutable file collections](#immutable-file-collections). 
+This release introduces 2 new APIs that facilitate configuring tasks already when they are created: APIs to create [tasks with constructor arguments](#create-tasks-with-constructor-arguments) and [immutable file collections](#immutable-file-collections).
```

Comment:
"already when they are created" => "at creation time"

Reply:
@lptr should we reword release-features.txt to say something other than "immutable tasks"?